### PR TITLE
Add system as OperationSuffix to initialize

### PR DESCRIPTION
**Pull request by fcbajao**
Resolves https://github.com/openbao/openbao/issues/407

## Target Release

1.14.7

### CODE REVIEW

**Comment by fcbajao** [comment]
cc @cipherboy

File: vault/logical_system_paths.go [hunk 1]
```diff
@@ -584,6 +584,7 @@ func (b *SystemBackend) configPaths() []*framework.Path {
 				logical.UpdateOperation: &framework.PathOperation{
 					DisplayAttrs: &framework.DisplayAttributes{
 						OperationVerb: "initialize",
```
Comment (cipherboy) [inline vault/logical_system_paths.go:586]: Just a note for others, but the subsequent place this is done (line 817 originally prior to this change) isn't affected as the entire group of operations has `OperationPrefix: "rekey-attempt"`, making the full function call `rekey_attempt_initialize(...)` in the Ruby client. 

Further this is the [only keyword](https://stackoverflow.com/questions/16099073/ruby-reserved-class-method-names) that appears to be reserved:

```
$ comm -12 /tmp/ruby-reserved-keywords /tmp/operations 
initialize
```

**Review by cipherboy — APPROVED** [review]

**Comment by fcbajao** [comment]
@cipherboy there are failing checks. Is there anything needed to be done on my end?

**Comment by cipherboy** [comment]
@fcbajao Ah, do you mind running `make fmt`? It [looks like](https://github.com/openbao/openbao/actions/runs/9944103236/job/27469857104?pr=408) formatting differs. A [changelog entry](https://github.com/openbao/openbao/blob/main/changelog/README.md) might be nice as well just for consistency, but I don't know that we consider OAS production ready yet, given we don't have any official client libraries built off of it.

**Comment by fcbajao** [comment]
> but I don't know that we consider OAS production ready yet, given we don't have any official client libraries built off of it.

@cipherboy if this is the case, maybe we can skip the changelog entry for now.

**Comment by fcbajao** [comment]
> Ah, do you mind running `make fmt`? It [looks like](https://github.com/openbao/openbao/actions/runs/9944103236/job/27469857104?pr=408) formatting differs

@cipherboy fixed the formatting. Please let me know if there's anything else needed on my end.

**Comment by cipherboy** [comment]
Given the above discussion around changelog, I'm going ahead and merging. Thanks @fcbajao!

**Pull request merged**